lib/src/base-styles/_nx-btn-bar.scss

```diff
@@ -8,5 +8,6 @@
 .nx-btn-bar {
   @include container-horizontal;
 
-  margin: 8px 0;
+  margin: $nx-spacing-sm 0;
+  text-align: right;
```

Comment:
This is a pretty big change, do we really want to be sneaking this in like this?

Reply:
> This is a pretty big change, do we really want to be sneaking this in like this?

I thought about that, it's first time that it has appeared in our refactoring so that's why I added it. I'm ok with pulling this out to a separate story and just accepting that in this case our work will not match the Zeplin file.

Reply:
> I thought about that, it's first time that it has appeared in our refactoring so that's why I added it. I'm ok with pulling this out to a separate story and just accepting that in this case our work will not match the Zeplin file.

The bigger question to me is whether we actually want this to be universally styled this way.  For instance it makes the `NxButton` examples arguably look weird

Reply:
> > This is a pretty big change, do we really want to be sneaking this in like this?
> 
> I thought about that, it's first time that it has appeared in our refactoring so that's why I added it. I'm ok with pulling this out to a separate story and just accepting that in this case our work will not match the Zeplin file.

I have reverted it here: https://github.com/sonatype/sonatype-react-shared-components/pull/75/commits/98ef407d49b7b11e031a02d8bc56aef15c1be9af